lib/ssl/doc/ssl_app.md

```diff
@@ -29,19 +29,24 @@ Erlang.
 For current statement of standards compliance see the
 [User's Guide](standards_compliance.md).
 
-## DEPENDENCIES
+## Dependencies
 
-The SSL application uses the `public_key`, `asn1` and Crypto application to
+The SSL application uses the `public_key`, `asn1` and `Crypto` application to
```

Comment:
We have internal guidelines for how to write application names:
http://otp.ericsson.se/internal/guidelines/documentation_guidelines/part_frame.html
http://otp.ericsson.se/product/internal/documentation/conventions/appl_names.shtml
And by that I guess it should be "the Public_Key, Asn1 and Crypto application"

Reply:
Fixed